docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1005,17 +1005,16 @@ Unit used to display time values.
 end::time[]
 
 tag::timeoutparms[]
-
-tag::timeout[]
 tag::master-timeout[]
 `master_timeout`::
-(Optional, <<time-units, time units>>) Specifies the period of time to wait for
+(Optional, <<time-units, time units>>) Time period to wait for
 a connection to the master node. If no response is received before the timeout
 expires, the request fails and returns an error. Defaults to `30s`.
 end::master-timeout[]
 
+tag::timeout[]
 `timeout`::
-(Optional, <<time-units, time units>>) Specifies the period of time to wait for
+(Optional, <<time-units, time units>>) Time period to wait for
 a response. If no response is received before the timeout expires, the request
 fails and returns an error. Defaults to `30s`.
 end::timeout[]
```

Comment:
Previously, these tags were pulling the `master_timeout` query param into several APIs that didn't support it.